### ISSUE TITLE
13 - Build and upload PM convergence maps to HuggingFace

### ISSUE THREAD

**Issue by Justinezgh** [Ciela-Institute/CosmOrford#13]
## Task

  - [ ] Write the script to build the kappa map from density planes and to generate the HF dataset.
  - [ ] Check the construction of kappa map by comparing power spectra.
  - [ ] Build the dataset and put it on rorqual `rrg-lplevass/shared/`

This dataset will be used as a new N-body reference for the emulator (issue #7 ) and as a training set for the compressor experiments.

**Comment by rouzib** [comment]
I've generated these two datasets:

| Dataset | Path | Cosmologies | Maps / Cosmo | Train Set | Test Set |
| :--- | :--- | :--- | :--- | :--- | :--- |
| **1024³** | `rrg-lplevass/shared/wl_chall_data/hf_pm_1024/` | 6,846 | 24 | 131,443 | 32,861 |
| **1560³ (Small)** | `rrg-lplevass/shared/wl_chall_data/hf_pm_1560/` | 104 | 24 | 1,996 | 500 |

The small 1560³ dataset is to assess if we need to run higher resolution PM simulations or if the 1024³ ones are enough.
